pom.client.xml

```diff
@@ -393,6 +393,10 @@
                 <title>Azure Storage - Blobs</title>
                 <packages>com.azure.storage.blob*</packages>
               </group>
+              <group>
+                <title>Azure Storage - Queues</title>
+                <packages>com.azure.storage.client.queue*</packages>
+              </group>
```

Comment:
I don't think you've generated the aggregate JavaDoc and reviewed it? Please do that as part of the pre-PR checks - because this isn't the package name for queues - you have an extra 'client' in there that would mean the 'Azure Storage - Queues' group will not have any content in it.

Reply:
Will do.

Reply:
It is there now. Thanks!